### PR TITLE
add a helm template to generate the label section

### PR DESCRIPTION
**Pull request by jiaqiluo**
issue: https://github.com/rancher/rancher/issues/31667

### CODE REVIEW

**Review by nickgerace — APPROVED** [review]

File: chart/templates/serviceAccount.yaml [hunk 1]
```diff
@@ -3,7 +3,4 @@ apiVersion: v1
 metadata:
   name: {{ template "rancher.fullname" . }}
   labels:
-    app: {{ template "rancher.fullname" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    heritage: {{ .Release.Service }}
-    release: {{ .Release.Name }}
+{{ include "rancher.labels" . | indent 4 }}
```
Comment (cbron) [inline chart/templates/serviceAccount.yaml:6]: nit: add newline at end of file

**Review by cbron — APPROVED** [review]

File: chart/templates/serviceAccount.yaml (hunk 1 above)
Comment (jiaqiluo) [reply chart/templates/serviceAccount.yaml]: fixed.

**Review by cbron — APPROVED** [review]

**Review by brendarearden — APPROVED** [review]
LGTM

**Review by nickgerace — APPROVED** [review]
Smash that merge button! @jiaqiluo

**Pull request merged**